control/controlbase/noiseexplorer_test.go

```diff
@@ -471,5 +441,3 @@ func RecvMessage(session *noisesession, message *messagebuffer) (*noisesession,
 	session.mc = session.mc + 1
 	return session, plaintext, valid
 }
-
-func main() {}
```

Comment:
This is really weird, I'm having trouble understanding why this was ever necessary.